<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriorityScheduling</class>
 <widget class="QWidget" name="PriorityScheduling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1922</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="inpeaceTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="outpeaceTable">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="outpeaceTable2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>491</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>700</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>手动添加进程</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="suspendLine">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>660</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="suspendButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>660</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>挂起</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="notsuspendLine">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>660</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="notsuspendButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>660</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>解挂</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>700</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>700</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_createButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>700</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>自动创建进程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>挂起队列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>就绪队列及正在运行进程</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>后备队列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopAuto_createButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>700</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>停止自动创建进程</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="cpuTable">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>380</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CPU_Priority Scheduling</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpu_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="num_label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="finshTable">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>400</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>380</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>完成队列</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="peaceTable">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>110</y>
     <width>761</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>    内存分区表：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
